packages/components/src/components/Combobox/Combobox.test.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import { Combobox } from "./Combobox";
+
+const SELECT_AN_OPTION_LABEL = "Select an option";
+
+describe("<Combobox />", () => {
+  const items = [
+    { label: "Item 1", value: "Item 1" },
+    { label: "Item 2", value: "Item 2" },
+    { label: "Item 3", value: "Item 3" },
```

Comment:
wouldnt it be better to test this with values that are different to the labels? to make sure that we dont mix them up

Reply:
Good idea! Done [here](https://github.com/Localitos/pluto/pull/1788/files#diff-054dc0d4d697e47e94abef3f9fd8ec006ccd0addf0a3996f5bb1b6609faf6b8dR10)